requirements.txt

```diff
@@ -2,8 +2,10 @@ keyboard
 mouse
 # pyaudio
 Pillow
-git+git://github.com/samclane/lifxlan@master#egg=lifxlan
+git+https://github.com/samclane/lifxlan@master#egg=lifxlan
 numexpr
 numpy
 mss
-pystray
\ No newline at end of file
+pystray
+pypiwin32
```

Comment:
I've read that `pywin32` is a newer version of `pypiwin32`. Is `pyaudio` only available on the latter?